src/main/java/frc/trigon/robot/utilities/CurrentWatcher.java

```diff
@@ -0,0 +1,53 @@
+package frc.trigon.robot.utilities;
+
+import edu.wpi.first.wpilibj.Notifier;
+import edu.wpi.first.wpilibj.Timer;
+
+import java.util.function.Supplier;
+
+/**
+ * A clas that checks if the current passed a certain current limit for a certain amount of time and if so, runs a runnable.
```

Comment:
class*
"checks if the current" maybe something like "the motor's current"
Maybe add a period after the word time, and remove the word "and"